aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -66,7 +85,6 @@ module AptosFramework::Account {
         currency_code_required: bool,
     ) {
         assert!(Signer::address_of(account) == @CoreResources, Errors::requires_address(ENOT_APTOS_FRAMEWORK));
-        move_to(account, Marker<T> {});
```

Comment:
just a question. what was the original intention for introducing marker?

Reply:
to support multi framework I think